notebooks/add_a_single_event_to_a_sketch.ipynb

```diff
@@ -0,0 +1,187 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Timesketch_add_a_single_event_to_a_sketch.ipynb",
+      "provenance": [],
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/google/timesketch/blob/master/notebooks/Timesketch_add_a_single_event_to_a_sketch.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "E2meYbWjdnyR"
+      },
+      "source": [
+        "# Setup\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "ZoKUE_mbwOyy"
+      },
+      "source": [
+        "# @title Install dependencies\n",
+        "# @markdown Only execute if not already installed and running a cloud runtime\n",
+        "!pip install -q timesketch_api_client\n",
+        "!pip install -q vt-py\n",
+        "!pip install -q nest_asyncio"
```

Comment:
why are these required? That is the vt-py and nest_asyncio???

also can you hide teh code in the forms?

Reply:
done